packages/form/components/Form.astro

```diff
@@ -22,12 +22,13 @@ const {
 
 const formTheme = theme ?? 'light';
 const formName = Array.isArray(formGroups) ? null : formGroups?.name || null;
+const formId = Array.isArray(formGroups) ? null : formGroups?.id || null;
```

Comment:
If `formGroups` is an array, generate a UUID for the `formId` instead of null.

```suggestion
const formId = Array.isArray(formGroups) ? uuid() : formGroups?.id || null;
```